src/main/java/com/stripe/model/reporting/ReportRun.java

```diff
@@ -0,0 +1,103 @@
+package com.stripe.model.reporting;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.ExpandableField;
+import com.stripe.model.FileUpload;
+import com.stripe.model.HasId;
+import com.stripe.model.MetadataStore;
+import com.stripe.model.StripeObject;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class ReportRun extends ApiResource implements HasId {
+  @Getter(onMethod = @__({@Override})) String id;
+  String object;
+  Long created;
+  String error;
+  Boolean livemode;
+  Parameters parameters;
+  String reportType;
+  FileUpload result;
+  String status;
+  Long succeededAt;
+
+  // <editor-fold desc="create">
+  /**
+   * Create a report run.
+   */
+  public static ReportRun create(Map<String, Object> params) throws StripeException {
+    return create(params, null);
+  }
+
+  /**
+   * Create a report run.
+   */
+  public static ReportRun create(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    return request(RequestMethod.POST, classUrl(ReportRun.class), params,
+      ReportRun.class, options);
+  }
+  // </editor-fold>
+
+  // <editor-fold desc="list">
+  /**
+   * List all report runs.
+   */
+  public static ReportRunCollection list(Map<String, Object> params) throws StripeException {
+    return list(params, null);
+  }
+
+  /**
+   * List all report runs.
+   */
+  public static ReportRunCollection list(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    return requestCollection(classUrl(ReportRun.class), params,
+      ReportRunCollection.class, options);
+  }
+  // </editor-fold>
+
+  // <editor-fold desc="retrieve">
+  /**
+   * Retrieve an report run.
+   */
+  public static ReportRun retrieve(String id) throws StripeException {
+    return retrieve(id, null);
+  }
+
+  /**
+   * Retrieve an report run.
+   */
+  public static ReportRun retrieve(String id, RequestOptions options) throws StripeException {
+    return retrieve(id, null, options);
+  }
+
+  /**
+   * Retrieve an report run.
+   */
+  public static ReportRun retrieve(String id, Map<String, Object> params,
+      RequestOptions options) throws StripeException {
+    return request(RequestMethod.GET, instanceUrl(ReportRun.class, id), params,
+      ReportRun.class, options);
+  }
+  // </editor-fold>
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public final class Parameters extends StripeObject {
+    Long intervalEnd;
+    Long intervalStart;
```

Comment:
I'm seeing quite a few additional fields as part of this one — is the idea to leave them out for now?

![image](https://user-images.githubusercontent.com/23105990/45122985-52219a00-b11a-11e8-8ac8-7f8a023ff11e.png)

Reply:
Ah yes that's what I get for trusting the fixture in the docs 🤦‍♂️, fixing.